src/components/resources/index.js

```diff
@@ -129,13 +129,13 @@ const RESOURCES = [
         title: "Wallet SDK Swift",
         Svg: Graphics.walletSdkSwift,
         content: "Build Swift-based applications using our SDKs",
-        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/"
+        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/documentation/atalaprismsdk/"
     },
     {
         title: "Wallet SDK TypeScript",
         Svg: Graphics.walletSdkTS,
         content: "Build web-based applications using our SDKs",
-        linkTo: "#"
+        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-ts/"
```

Comment:
Please explain the reason of this link change. So we are sure it is not breaking something somewhere else.

Reply:
reverted back to original link, we will do a link check in second iteration.

Reply:
# must have been a placeholder from early days before the repo was actually ready :)
This doesn't lead to anywhere and changing it to a good link does not incur a risk of breaking.